src/hotkey.rs

```diff
@@ -4,37 +4,42 @@ use std::ptr;
 use std::thread;
 
 use winapi::um::winuser::{
-    DispatchMessageW, GetMessageW, MessageBoxW, RegisterHotKey, TranslateMessage, MB_OK, MOD_ALT,
-    MOD_CONTROL, MOD_NOREPEAT, WM_HOTKEY,
+    DispatchMessageW, GetMessageW, MessageBoxW, RegisterHotKey, TranslateMessage,
+    GetKeyboardLayout, VkKeyScanExA, MB_OK, MOD_ALT, MOD_CONTROL, MOD_SHIFT, 
+    MOD_NOREPEAT, WM_HOTKEY,
 };
 
 use crate::Message;
 use crate::CHANNEL;
+use crate::config;
 
 pub fn spawn_hotkey_thread() {
+    let mut hotkey: Vec<String> = config::load_config().hotkey
+                                .split("+")
```

Comment:
Use `'+'` single quote to specify as `char` since we are only passing a single character